src/main.cpp

```diff
@@ -104,13 +121,14 @@ void setup()
 
     //define the sketch as anchor. It will be great to dynamically change the type of module
     DW1000Ranging.attachNewRange(newRange);
-    DW1000Ranging.attachBlinkDevice(newBlink);
+    DW1000Ranging.attachNewDevice(newDevice);
     DW1000Ranging.attachInactiveDevice(inactiveDevice);
     //Enable the filter to smooth the distance
     //DW1000Ranging.useRangeFilter(true);
 
-    //we start the module as an anchor
-    // DW1000Ranging.startAsAnchor("82:17:5B:D5:A9:9A:E2:9C", DW1000.MODE_LONGDATA_RANGE_ACCURACY);
+    //we start the module as a tag
+    DW1000Ranging.startAsTag("7C:00:22:EA:82:60:3B:9C", DW1000.MODE_SHORTDATA_FAST_LOWPOWER, false);
+    //DW1000.enableManualLedBlinking();
```

Comment:
We already have it in the block below. Just modify the 

```c
#elif defined(UWB_TAG)
    DW1000Ranging.startAsTag(ADDR_TAG, DW1000.MODE_LONGDATA_RANGE_LOWPOWER, false);
#endif
```

instead of adding more.

Reply:
then we can build a tag firmware using

```
pio run -e esp32_tag
```

or 

```
pio run -e esp32_anchor
```

otherwise you're hardcoding the role of this code.